content/swift/concepts/arrays/terms/isEmpty/isEmpty.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '.isEmpty'
+Description: 'Returns true if an array contains no elements, returns false if the array contains one or more elements.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.isEmpty`** method returns true if an array contains no elements, returns false if the array contains one or more elements.
+
+## Syntax
+
+```pseudo
+arrayName.isEmpty
+```
```

Comment:
There should be a sentence after the syntax block explaining what `arrayName` is.